testssl.sh

```diff
@@ -18412,10 +18413,14 @@ determine_service() {
           # no STARTTLS.
           determine_optimal_sockets_params
           determine_optimal_proto
+          BASIC_AUTH_HEADER=""
```

Comment:
I would rather see that in the top please like all other vars, see _Following variables make use of $ENV and can be used like_ , maybe below ``ASSUME_HTTP``

Reply:
> BASIC_AUTH_HEADER needs to be in the very beginning please (_Initialization part, further global vars just_)

Actually, it seems that BASIC_AUTH_HEADER is only used within `determine_service()`. So, it should be defined as a local variable in that function. By [convention](https://github.com/drwetter/testssl.sh/wiki/Coding-Style), local variables should be lower case. So, the above line can be deleted, and a line should be added  at the beginning of `determine_service()`: `local basic_auth_header=""`

Reply:
ACK